src/Bootstrappers/BroadcastingConfigBootstrapper.php

```
@@ -64,10 +72,35 @@ public function bootstrap(Tenant $tenant): void
+                $tenantBroadcastManager->extend($driver, $closure);
+            }
+
+            return $tenantBroadcastManager;
         });
+
+        // Swap currently bound Broadcaster instance for one that's resolved through the tenant BroadcastManager.
+        // Note that updating broadcasting config (credentials) in tenant context doesn't update the credentials
+        // used by the bound Broadcaster instance. If you need to e.g. send a notification in response to
+        // updating tenant's broadcasting credentials in tenant context, it's recommended to
+        // reinitialize tenancy after updating the credentials.
+        $this->app->extend(Broadcaster::class, function (Broadcaster $broadcaster) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$broadcaster'. (undefined)

(UnusedFormalParameter)